docs/services/endlessh.md

```diff
@@ -0,0 +1,70 @@
+# Endlessh
+
+[Endlessh-go](https://github.com/shizunge/endlessh-go) is a A golang implementation of endlessh, a ssh trapit. Installing it is powered by the [mother-of-all-self-hosting/ansible-role-endlessh](https://github.com/mother-of-all-self-hosting/ansible-role-endlessh) Ansible role.
+
+## Dependencies
+
+This service requires the following other services:
+
+- (optionally) [Traefik](traefik.md) - a reverse-proxy server for exposing endlessh publicly
+- (optionally) [Prometheus](./prometheus.md) - a database for storing metrics
+- (optionally) [Grafana](./grafana.md) - a web UI that can query the prometheus datasource (connection) and display the logs
```

Comment:
I suppose a valid pre-requisite for any sort of SSH tarpit system is to:

- either: not have an SSH server at all (likely not the case for many)
- or: relocating the real SSH server to another port (freeing up the default port `22` for the tarpit server)

We should probably indicate this somewhere. Maybe in a new "Prerequisites" section, because it's not really a dependency.